.github/workflows/breaking-change-review.yml

```diff
@@ -0,0 +1,145 @@
+
+      - name: Check QA approvals
+        if: steps.check_label.outputs.result == 'true'
+        id: check_approvals
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const { data: reviews } = await github.rest.pulls.listReviews({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.issue.number,
+            });
+            
+            const mobileQA = '${{ steps.set_reviewers.outputs.mobile_qa }}'.split(',');
+            const desktopQA = '${{ steps.set_reviewers.outputs.desktop_qa }}'.split(',');
+            
+            const mobileQAApproved = reviews.some(review => 
+              review.state === 'APPROVED' && mobileQA.includes(review.user.login)
+            );
+            const desktopQAApproved = reviews.some(review => 
+              review.state === 'APPROVED' && desktopQA.includes(review.user.login)
+            );
+            
+            console.log(`Mobile QA approved: ${mobileQAApproved}`);
+            console.log(`Desktop QA approved: ${desktopQAApproved}`);
+            
+            core.setOutput('mobile-qa-approved', mobileQAApproved);
+            core.setOutput('desktop-qa-approved', desktopQAApproved);
+            
+            return mobileQAApproved && desktopQAApproved;
+
+      - name: Block PR if conditions not met
+        if: steps.check_label.outputs.result == 'true' && steps.check_approvals.outputs.result != 'true'
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const mobileQAApproved = ${{ steps.check_approvals.outputs.mobile-qa-approved }};
+            const desktopQAApproved = ${{ steps.check_approvals.outputs.desktop-qa-approved }};
+            
+            let message = 'This PR has the breaking change label and requires approval from both mobile-qa and desktop-qa teams before it can be merged.\n\n';
+            
+            if (!mobileQAApproved && !desktopQAApproved) {
+              message += 'Both mobile-qa and desktop-qa teams have not approved this PR yet.';
+            } else if (!mobileQAApproved) {
+              message += 'The mobile-qa team has not approved this PR yet.';
+            } else if (!desktopQAApproved) {
+              message += 'The desktop-qa team has not approved this PR yet.';
+            }
+            
+            await github.rest.checks.create({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              name: 'QA Approval Check',
+              head_sha: context.payload.pull_request.head.sha,
+              status: 'completed',
+              conclusion: 'failure',
+              output: {
+                title: 'QA Approval Required',
+                summary: message
+              }
+            });
+
+      - name: Allow PR merge if conditions are met
```

Comment:
Do we need the bot approval at all, if we have the status check already? It's kinda duplication 🤔